remote-access/web-server/nginx.md

```diff
@@ -130,3 +130,76 @@ Add some dynamic PHP content by replacing the current content:
 ```
 
 Save and refresh your browser. You should see a page with the PHP version, logo and current configuration settings.
+
+What next! - I'll be adding a guide for using Lets Encrypt to add ssl to the Raspberry Pi nginx server.
+
+In this tutorial, I’ll guide you through the process of installing let’s encrypt SSL certificates on your nginx powered website. By the end of the tutorial, we’ll have done the following:
+
+Installed the let’s encrypt service
+Generate a free let’s encrypt SSL certificate
+Install free SSL certificate in nginx to secure your site
+Secure site with SSL
+
+Throughout this tutorial, I’ll be installing and configuring in Ubuntu 16.04, but the instructions will also work fine with Debian. I haven’t tested but there’s no reason why this tutorial won’t work to install SSL to your nginx site on Raspbian for a Raspberry Pi.
```

Comment:
"I’ll be installing and configuring in Ubuntu 16.04..." - the documentation in this repo is written first and foremost for Raspbian, as that's the only officially-supported Raspberry Pi OS.